support-workers/src/main/scala/com/gu/zuora/subscriptionBuilders/GuardianLightSubscriptionBuilder.scala

```diff
@@ -37,3 +35,8 @@ class GuardianLightSubscriptionBuilder(
     subscribeItemBuilder.build(subscriptionData, state.salesForceContact, Some(state.paymentMethod), None)
   }
 }
+
+object GuardianLightSubscriptionBuilder {
+  // The user isn't charged until day 15
+  val gracePeriodInDays = 15
```

Comment:
Pulled this out to make it easy to share with the code which builds the email fields.